Callbacks/UserSettingCallback.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Bugsnag\BugsnagBundle\Callbacks;
+
+use Bugsnag\Report;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class UserSettingCallback
+{
+    /**
+     * The token resolver.
+     *
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface|null
+     */
+    protected $tokens;
+
+    /**
+     * The auth checker.
+     *
+     * @var \Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface|null
+     */
+    protected $checker;
+
+    /**
+     * @var bool
+     */
+    protected $setUser;
+
+    /**
+     * @param null|\Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface $tokens
+     * @param null|\Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface        $checker
+     * @param bool                                                                                     $setUser
+     *
+     * @return void
+     */
+    public function __construct(
+        TokenStorageInterface $tokens = null,
+        AuthorizationCheckerInterface $checker = null,
+        $setUser = true
+    ) {
+        $this->tokens = $tokens;
+        $this->checker = $checker;
+        $this->setUser = $setUser;
+    }
+
+    /**
+     * Define a callback to set the currently authenticated user as the user
+     * on any Bugsnag reports that are sent.
```

Comment:
Please change to:

```php
    /**
     * Define a callback to set the currently authenticated user.
     *
     * @param \Bugsnag\Report $report
     *
     * @return void
     */
```

This is because "short descriptions" cannot span more than 1 line, and we also apply the extra cs constraint that they cannot be longer than 80 characters.